pkg/targets/adapter/splunktarget/adapter.go

```diff
@@ -121,7 +121,9 @@ func newClient(hecURL url.URL, hecToken, index, hostname string, skipTLSVerify b
 		Transport: httpTransport,
 	}
 
-	hecURL.Path = eventURLPath
+	if hecURL.Path == "" || hecURL.Path == "/" {
```

Comment:
why are we expecting the `/` ?

Reply:
A user might write the Splunk URL like:

- https://my.splunk.url:8888/
- https://my.splunk.url:8888

both cases will make the adapter append the default Splunk path, resulting in:

https://my.splunk.url:8888/services/collector/event/1.0

whereas any other path like https://my.splunk.url:8888/mypath will keep the user's path and won't append any suffix.